src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -157,6 +157,7 @@ export class QuerySuggestPreview extends Component implements IComponentBindings
   private buildQuery(searchQuery: string): IQuery {
     const { searchHub, pipeline, tab, locale, timezone, context } = this.queryController.getLastQuery();
     return {
+      firstResult: 0,
```

Comment:
Rather than destructuring, I think it's better to return the entire last query with the two values overridden. This would have avoided that last two jiras and maybe future ones in this area. e.g.

```
const lastQuery = this.queryController.getLastQuery();
return {
    ...lastQuery,
    q: searchQuery,
    numberOfResults: this.options.numberOfPreviewResults
}
```

Reply:
🤔 What you're proposing is to put go the previous error instead of the current one.
Destructuring is fine we just forgot one property.
https://github.com/coveo/search-ui/pull/1292